examples/getting-started-movielens/03a-Training-with-TF.ipynb

```diff
@@ -635,6 +633,26 @@
     "# print validation loss\n",
     "history.history"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "INFO:tensorflow:Assets written to: /model/movielens_tf/1/model.savedmodel/assets\n"
+     ]
+    }
+   ],
+   "source": [
+    "MODEL_NAME_TF = os.environ.get(\"MODEL_NAME_TF\", \"movielens_tf\")\n",
+    "MODEL_PATH_TEMP_TF = os.path.join(MODEL_BASE_DIR, MODEL_NAME_TF, \"1/model.savedmodel\")\n",
+    "\n",
+    "model.save(MODEL_PATH_TEMP_TF)"
```

Comment:
We should base this off of INPUT_DATA_DIR so that the unittests can pass

Reply:
done